lib/Common/Memory/RecyclerTelemetryInfo.h

```diff
@@ -111,8 +112,7 @@ namespace Memory
         Js::Tick recyclerStartTime;
         bool inPassActiveState;
 
-        // TODO: update list below to SList.  Need to ensure we have same allocation semantics (specifically heap allocs, no exceptions on failure)
-        RecyclerTelemetryGCPassStats* lastPassStats;
+        SList<RecyclerTelemetryGCPassStats, HeapAllocator> gcPassStats;
```

Comment:
Just a nitpick but why not add a typedef for this type of list?

Reply:
good suggestion, will do.